lib/event_tracker/trackers/base.rb

```diff
@@ -0,0 +1,13 @@
+module EventTracker
+  module Trackers
+    class Base
+      def initialize(doer_id)
+        @doer_id = doer_id
+      end
+
+      def track(event_name, event_label, properties)
+        raise 'Implement in adapter'
```

Comment:
http://ruby-doc.org/core-2.3.0/NotImplementedError.html
```ruby
raise NotImplementedError.new('Implement in adapter')
```
could be a more accurate subclass of `Exception`